### DISCUSSION TITLE
Failed to start opa engine

### DISCUSSION THREAD

**Discussion by Handontsmoke** [opentdf/platform#1377] [Platform Q&A]
Hello,

I am trying to set up OpenTDF . I have followed the installation steps, but I am encountering an error when I run the command
`docker run --network opentdf_platform -p "127.0.0.1:8080:8080" -v "$(pwd)/kas-keys/:/keys/" -v "$(pwd)/opentdf.yaml:/home/nonroot/.opentdf/opentdf.yaml" -it registry.opentdf.io/platform:nightly start`

Here is the relevant part of the output:
![image](https://github.com/user-attachments/assets/0fb33775-aba9-4ede-a092-ea360df21a11)
failed to read config file: open ./opentdf-opa.yaml: no such file or directory
Do I miss some configuration files?

**Comment by jrschumacher** [comment]
We [no longer bundle OPA](https://github.com/opentdf/platform/pull/1048) with OpenTDF due to performance reasons. We still use rego but don't currently support custom rego within the platform.

Our concept of an Entity Resolution Service is where you might want to run custom rego or call the OPA cluster running within your environment.

**Reply by jrschumacher** [reply]
What is your intention for using OPA?

**Reply by Handontsmoke** [reply]
I just want to run this command
`docker run --network opentdf_platform -p "127.0.0.1:8080:8080" -v "$(pwd)/kas-keys/:/keys/" -v "$(pwd)/opentdf.yaml:/home/nonroot/.opentdf/opentdf.yaml" -it registry.opentdf.io/platform:nightly start`
but it seems like the container is automatically starting the OPA engine. How can I avoid this or modify it?
